[Update  to Quarkus 2.1.1.Final] Throw java.io.IOException when press 'w' in DevConsole

### Describe the bug

```log
2021-08-08 18:16:24,594 ERROR [io.qua.ver.htt.dep.dev.con.DevConsoleProcessor] (Aesh InputStream Reader) Failed to launch browser: java.io.IOException: Cannot run program "rundll32 url.dll,FileProtocolHandler": CreateProcess error=2
, 系统找不到指定的文件。
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
        at java.base/java.lang.Runtime.exec(Runtime.java:589)
        at java.base/java.lang.Runtime.exec(Runtime.java:448)
        at io.quarkus.vertx.http.deployment.devmode.console.DevConsoleProcessor.openBrowser(DevConsoleProcessor.java:457)
        at io.quarkus.vertx.http.deployment.devmode.console.DevConsoleProcessor.lambda$setupConsole$1(DevConsoleProcessor.java:427)
        at io.quarkus.deployment.console.ConsoleStateManager$1.accept(ConsoleStateManager.java:73)
        at io.quarkus.deployment.console.ConsoleStateManager$1.accept(ConsoleStateManager.java:46)
        at io.quarkus.deployment.console.AeshConsole.lambda$setup$1(AeshConsole.java:199)
        at org.aesh.terminal.EventDecoder.accept(EventDecoder.java:118)
        at org.aesh.terminal.EventDecoder.accept(EventDecoder.java:31)
        at org.aesh.terminal.io.Decoder.write(Decoder.java:133)
        at org.aesh.readline.tty.terminal.TerminalConnection.openBlocking(TerminalConnection.java:216)
        at org.aesh.readline.tty.terminal.TerminalConnection.openBlocking(TerminalConnection.java:203)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.io.IOException: CreateProcess error=2, 系统找不到指定的文件。
        at java.base/java.lang.ProcessImpl.create(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:478)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:154)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1107)
        ... 16 more

```

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

Microsoft Windows [ 10.0.19043.1110]

### Output of `java -version`

java version "11.0.7" 2020-04-14 LTS Java(TM) SE Runtime Environment 18.9 (build 11.0.7+8-LTS) Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.7+8-LTS, mixed mode)

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

2.1.1.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/console/DevConsoleProcessor.java
+++ b/extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/console/DevConsoleProcessor.java
@@ -495,7 +495,7 @@ public class DevConsoleProcessor {
                     rt.exec(new String[] { "xdg-open", url });
                     break;
                 case WINDOWS:
-                    rt.exec(new String[] { "rundll32 url.dll,FileProtocolHandler", url });
+                    rt.exec(new String[] { "rundll32", "url.dll,FileProtocolHandler", url });
                     break;
                 case OTHER:
                     log.error("Cannot launch browser on this operating system");

